docs/get-started.md

```diff
@@ -0,0 +1,658 @@
+  "mappings": [
+    {
+      "endpoint": "/group_data/double_endpoint",
+      "type": "double",
+      "explicit_timestamp": false
+    },
+    {
+      "endpoint": "/group_data/string_endpoint",
+      "type": "string",
+      "explicit_timestamp": false
+    }
+  ]
+}
+```
+
+And finally the definition of the property interface:
+
+```json
+{
+  "interface_name": "org.astarte-platform.rust.get-started.Property",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "properties",
+  "ownership": "device",
+  "description": "Property interface for the get-started of the Astarte device SDK for the Rust programming language.",
+  "mappings": [
+    {
+      "endpoint": "/double_endpoint",
+      "type": "double",
+      "allow_unset": true
+    }
+  ]
+}
+```
+
+To install the three interfaces in the Astarte instance, open the Astarte dashboard, navigate to the
+interfaces tab and click on install new interface. You can copy and paste the JSON files for each
+interface in the right box overwriting the default template.
+
+### Registering the device
+
+Devices should be pre-registered to Astarte before their first connection. With the Astarte device
+SDK for Rust this can be achieved in two ways:
+
+- By registering the device on Astarte manually, obtaining a credentials secret and transferring it
+  on the device.
+- By using the included registration utilities provided by the SDK. Those utilities can make use of
+  a registration JWT issued by Astarte and register the device automatically before the first
+  connection.
+
+To keep this guide as simple as possible we will use the first method, as a device can be registered
+using the Astarte dashboard with a couple of clicks.
+
+To install a new device start by opening the dashboard and navigate to the devices tab. Click on
+register a new device, there you can input your own device ID or generate a random one. For example
+you could use the device ID `2TBn-jNESuuHamE2Zo1anA`. Click on register device, this will register
+the device and give you a credentials secret. The credentials secret will be used by the device to
+authenticate itself with Astarte. Copy it somewhere safe as it will be used in the next steps.
+
+## Creating a Rust project
+
+First, make sure you have Rust v1.78.0 or later installed, since it is the MSRV of the Astarte
+device Rust SDK. Then, create a new project:
+
+```bash
+cargo new astarte-rust-project
+```
+
+Then, we need to add the following dependencies in the Cargo.toml file:
+
+- `astarte-device-sdk`, to properly use the Astarte SDK
+- `tokio`, a runtime for writing asynchronous applications
+- `serde` and `serde_json`, for serializing and deserializing Rust data structures. They are useful
+  since we want to retrieve some device configuration stored in a json file.
+
+We also suggest you to add the following dependencies
+
+- `tracing` and `env_logger`, for a better logging experience
+
+Thus, the content of the Cargo.toml file will:
```

Comment:
```suggestion
You can run the following cargo add command:

```sh
cargo add astarte-device-sdk --features=derive
cargo add tokio --features=full
cargo add serde serde-json --features=serde/derive
cargo add tracing tracing-subscriber
cargo add eyre
```

Thus, the content of the Cargo.toml file will:
```

The github comment is broken :^(

Reply:
This will use the latest available version of the libraries